### ISSUE TITLE
broken invite link in E-Mail

### ISSUE THREAD

**Issue by jjohanning** [directus/directus#14278]
### Describe the Bug

The Link in the E-Mail is broken.

Link: https://my-internal-domain/admin/accept-invite?%0Dtoken=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJlbWFpbCI6Impqb2hhbm5pbmdAZ2%0D1haWwuY29tIiwic2NvcGUiOiJpbnZpdGUiLCJpYXQiOjE2NTY5MjIxNDksImV4cCI6MTY1NzUyN%0Djk0OSwiaXNzIjoiZGlyZWN0dXMifQ.hdcRCh-8xiEEZG_RAYj4z-aYbS9zFzmS_l41LrN20Ik

When I remove the %0D behind the "?"  in the above url, the link is ok. 

### To Reproduce

Invite a new user

### What version of Directus are you using?

9.12.2

### What version of Node.js are you using?

16.15.1

### What database are you using?

Postgresql 13

### What browser are you using?

Chrome

### How are you deploying Directus?

Docker

**Comment by emrahnazif** [comment]
This works as expected in v9.13.0.
@jjohanning Can you check your project URL settings from `/admin/settings/project`?

If you have `/` at the end, please remove it and try again.

**Comment by emrahnazif** [comment]
it is possible that this can be related Environment Variables in your .env settings. 

https://docs.directus.io/self-hosted/config-options/#type-casting-and-nesting

According to doc:
The `PUBLIC_URL` value is used for things like OAuth redirects, forgot-password emails, and logos that needs to be publicly available on the internet.

**Comment by jjohanning** [comment]
removed the / from PUBLIC_URL no luck. I think the / at the end of the url is not effecting the parameters of the url behing the ?

**Comment by jjohanning** [comment]
same problem with version 9.13.0 :-(

**Comment by emrahnazif** [comment]
> same problem with version 9.13.0 :-(

@jjohanning after your v9.13.0 comment. I tried it again that maybe I missed something. 
It worked fine again for me with v9.13.0. I used different user role this time. 

This shouldn't change anything but I use SMTP settings for my emails. 

Also in your case the `accept-invite?%0Dtoken=*****` carriage return is between `accept-invite` and `token`. I am not sure what can cause at it. 

Maybe someone else can also try and share their result here...
